### PR TITLE
Update vocabularies.xml

### PR DESCRIPTION
**Pull request by meliezer**
The current version of dataset_type.xml is older than dataset_type_2015-07-10.xml, so it doesn't have samplingEvent which was added to
http://rs.gbif.org/vocabulary/gbif/dataset_type_2015-07-10.xml
I don't know why, but either the link should be changed here, or dataset_type.xml be updated.

Cheers,
Menashè

### CODE REVIEW

**Comment by MattBlissett** [comment]
Thanks, @meliezer.

**Comment by meliezer** [comment]
Thanks @MattBlissett . May I ask you how could I hack it manually in my current IPT installation? I would also like to have always http://rs.gbif.org/vocabulary/gbif/dataset_subtype.xml added by default to keywords page, and if possible hack also the Core mapping GUI so I see only Event, and not the Occurrence Core.

Cheers,
Menashè

**Comment by MattBlissett** [comment]
That file (`src/test/resources/vocabularies/vocabularies.xml`) is only used in unit tests, during development of the IPT.

Where do you miss the type?

image

(The subtype is just below the type box.)

image

**Comment by meliezer** [comment]
@MattBlissett, let me rephrase. I wanted to change the file in order to solve issues in manage/metadata-keywords.do, but it seems not related, so I can create issues for:
1. In the keywords page I see by default
GBIF Dataset Type Vocabulary: http://rs.gbif.org/vocabulary/gbif/dataset_type.xml
and I need to change it to
GBIF Dataset Type Vocabulary: http://rs.gbif.org/vocabulary/gbif/dataset_type_2015-07-10.xml
Since I was samplingEvent as a keyword. I don't know why it's missing from dataset_type.xml.

2. I always need to add also 'GBIF Dataset Subtype Vocabulary: http://rs.gbif.org/vocabulary/gbif/dataset_subtype.xml', so I thought to hack and see it always.

3. Not related, but in the dropdown list in 'Darwin Core Mappings' I prefer to not see anymore the Occurrence Core option, if possible.

Thank you for your help!

**Comment by MattBlissett** [comment]
https://github.com/gbif/ipt/blob/master/src/main/java/org/gbif/ipt/service/manage/impl/ResourceManagerImpl.java#L2330

The subtype is added in this method. It looks like it depends on the resource having a type and subtype set, on the main "Basic Metadata" page.
